mungegithub/Dockerfile-publisher

```diff
@@ -15,9 +15,9 @@
 FROM google/debian:jessie
 MAINTAINER Chao Xu <xuchao@google.com>
 RUN apt-get update \
- && apt-get install -y -qq git=1:2.1.4-2.1+deb8u2 \
- && apt-get install -y -qq mercurial=3.1.2-2+deb8u3 \
- && apt-get install -y -qq ca-certificates wget jq vim tmux \
+ && apt-get install -y -qq git=1:2.1.4-2.1+deb8u4 \
+ && apt-get install -y -qq mercurial \
+ && apt-get install -y -qq ca-certificates wget jq vim tmux bsdmainutils tig \
```

Comment:
A marker for myself. What is `bsdmainutils` util for?

Reply:
For binary search in sorted files: http://linuxcommand.org/lc3_man_pages/look1.html